src/setupTests.js

```diff
@@ -6,6 +6,8 @@ import '@testing-library/jest-dom';
 
 import mswServer from 'api-mocks/msw-server';
 
+globalThis.IS_REACT_ACT_ENVIRONMENT = true;
```

Comment:
Where is this referenced? One of the testing libraries?

Edit: yes https://react.dev/blog/2022/03/08/react-18-upgrade-guide

Reply:
yep, was part of the official upgrade docs. I find it odd that I can't find this in their CRA template, but yeah :shrug: